### ISSUE TITLE
Show `<unchanged>` for empty cells in `ApplicationProtocol`, `SimulationTime`, `SimulationTimeUnit`? 

### ISSUE THREAD

**Issue by PavelBal** [esqLABS/shinyScenarioEditor#19]
Empty cells in the columns

- `ApplicationProtocol`,
- `SimulationTime`
- `SimulationTimeUnit`
- `OutputPathsIds`

denote that the original values from the pkml will be taken. Can we show something like `<unchanged>` (grey color?) for these cells, so it is clear to the user that he does not have to fill those cells out?

**Comment by AKostiv8** [comment]
@PavelBal Yes, it is doable. Also we can consider restricting these cells to a read-only state

**Comment by PavelBal** [comment]
Read-only is not required, the user should be able to change them (but also leave empty to use the information from the pkml file)
